app/config/config.yml

```diff
@@ -210,7 +210,7 @@ scheb_two_factor:
         template: "@WallabagUser/Authentication/form.html.twig"
         mailer: Wallabag\UserBundle\Mailer\AuthCodeMailer
 
-kphoen_rulerz:
+rulerz:
```

Comment:
same here, wallabag rulerz bundle is based on the master from rulerz bundle, that has only a single commit on to if 0.x, that do this change: https://github.com/K-Phoen/RulerZBundle/commit/c9dc901c32cc6049df710dfeb8ceef01153ea3f1#diff-f5f83cd60e81d7bfdde9e349048d2089be00e9173cd2a8eb3af75c426aab8f48R70